index.js

```diff
@@ -0,0 +1,150 @@
+// ЗАДАЧИ логические операторы
+
+// Проверка значения из диапазона
+
+if (age >= 14 && age <= 90);
+
+// Проверка значения вне диапазона
+if (!(age >= 14 && age <= 90));
+
+if (age < 14 || age > 90);
+
+// ПРОВЕРКА ЛОГИНА
+let login = prompt("Кто там?", "");
+if (login == "Admin") {
+  console.log("");
+  let password = prompt("пароль?", "");
+  if (password == "я главный") {
+    console.log("Здравствуйте");
+  } else if (password == null) {
+    console.log("отменено");
+  } else {
+    console.log("неверный пароль");
+  }
+} else if (login == null) {
```

Comment:
wrong check

please check this if, it won't work

Reply:
fixed